### PR TITLE
Use a test-specific schema in `prism.test_core`

### PR DESCRIPTION
**Pull request by jacoblurye**
...so that no "generic" prism module tests depend on the CIDC data model.

### CODE REVIEW

File: cidc_schemas/prism/core.py [hunk 1]
```diff
@@ -578,7 +583,7 @@ def prismify(
         template.schema.get("prism_template_root_object_schema")
         or "clinical_trial.json"
     )
-    root_ct_schema = load_and_validate_schema(root_ct_schema_name)
+    root_ct_schema = load_and_validate_schema(root_ct_schema_name, schema_root)
```
Comment (curlup) [inline cidc_schemas/prism/core.py:586]: What do you think on removing `schema_root` default value in `load_and_validate_schema`

File: tests/prism/test_core.py [hunk 2]
```diff
@@ -275,11 +290,11 @@ def test_prism_local_files_format_multiple_extensions(monkeypatch):
                                     "merge_pointer": "artifact",
                                     "gcs_uri_format": {
                                         "format": "lambda val, ctx: 'subfolder/' + ctx['record'] + '/artifact.' + val.rsplit('.', 1)[-1]",
-                                        "check_errors": "lambda val: f'Bad file type {val!r}. It should be in one of .tiff .tif .qptiff .svs formats' if val.rsplit('.', 1)[-1] not in ['svs', 'tiff', 'tif', 'qptiff'] else None",
+                                        "check_errors": f"lambda val: f'{error_str}' if val.rsplit('.', 1)[-1] not in ['svs', 'tiff', 'tif', 'qptiff'] else None",
```
Comment (curlup) [inline tests/prism/test_core.py:293]: IMO double-formatting `f" f'{}' "` is a little bit hard to grasp.

File: tests/prism/test_core.py [hunk 3]
```diff
@@ -605,51 +549,48 @@ def test_prism_many_artifacts_from_process_as_on_one_record(monkeypatch):
...
     assert len(errs) == 0
 
     local_paths = [e.local_path for e in file_maps]
-    uuids = [e.upload_placeholder for e in file_maps]
+    upload_uuids = [e.upload_placeholder for e in file_maps]
 
-    assert 3 * 3 * 2 == len(
+    assert 3 * 2 * 2 == len(
         file_maps
     )  # (3 files * 3 fields from each record) * 2 records
```
Comment (curlup) [inline tests/prism/test_core.py:565]:
```suggestion
    )  # (3 files * 2 fields from each record) * 2 records
```

File: tests/prism/test_core.py [hunk 4]
```diff
@@ -605,51 +549,48 @@ def test_prism_many_artifacts_from_process_as_on_one_record(monkeypatch):
...
-    assert 3 * 3 * 2 == len(
-        set(uuids)
+    assert 3 * 2 * 2 == len(
+        set(upload_uuids)
     )  # (3 files * 3 fields from each record) * 2 records
```
Comment (curlup) [inline tests/prism/test_core.py:568]:
```suggestion
    )  # (3 files * 2 fields from each record) * 2 records
```

File: cidc_schemas/prism/core.py (hunk 1 above)
Comment (jacoblurye) [reply cidc_schemas/prism/core.py:586]: Fine by me!

Comment (jacoblurye) [reply cidc_schemas/prism/core.py:586]: There are a number of places in the code unrelated to this PR that rely on that default, so I think we should keep it for now.

**Pull request merged**